vscode/package.json

```diff
@@ -403,33 +410,62 @@
           "scope": "window",
           "order": 5
         },
-        "konveyor.labelSelector": {
+        "konveyor.analysis.labelSelector": {
           "type": "string",
           "default": "discovery",
           "description": "Expression to select incidents based on custom variables. Example: (!package=io.konveyor.demo.config-utils)",
           "scope": "window",
           "order": 6
         },
-        "konveyor.analyzeKnownLibraries": {
+        "konveyor.analysis.analyzeKnownLibraries": {
           "type": "boolean",
           "default": false,
           "description": "Whether analysis should include known open-source libraries",
           "scope": "window",
           "order": 7
         },
-        "konveyor.analyzeDependencies": {
+        "konveyor.analysis.analyzeDependencies": {
           "type": "boolean",
           "default": true,
           "description": "Whether analysis should include dependencies",
           "scope": "window",
           "order": 8
         },
-        "konveyor.analyzeOnSave": {
+        "konveyor.analysis.analyzeOnSave": {
           "type": "boolean",
           "default": true,
           "description": "Whether analysis of file should be run when saved",
           "scope": "window",
           "order": 9
+        },
+        "konveyor.diffEditorType": {
+          "type": "string",
+          "default": "diff",
+          "description": "Diff editor to use when resolving proposed solutions",
+          "scope": "window"
+        },
+        "konveyor.kai.backendURL": {
+          "type": "string",
+          "default": "0.0.0.0:8080",
+          "description": "Kai Backend URL",
+          "scope": "window"
+        },
+        "konveyor.kai.providerName": {
+          "type": "string",
+          "default": "ChatIBMGenAI",
+          "description": "URL for the Kai solution server",
+          "scope": "window"
+        },
+        "konveyor.kai.providerArgs": {
+          "type": "object",
+          "default": {
+            "model_id": "meta-llama/llama-3-1-70b-instruct",
+            "parameters": {
+              "max_new_tokens": 2048
+            }
```

Comment:
During my tests I need this to be:
```suggestion
            "model": "gpt-3.5-turbo"
       
```
Else I get errors like:

```Encountered errors: ["Completions.create() got an unexpected keyword argument 'model-id'"]```

Reply:
Hopefully these model configs don’t become a moving target. I do get the feeling it’ll end up a generic JSON “clob” to handle variations between models on the far back end.